src/prometheus_alert_rules/vault_sealed.rule

```diff
@@ -0,0 +1,8 @@
+alert: VaultSealedRule
+expr: vault_core_unsealed == 0
+for: 0m
+labels:
+    severity: critical
+annotations:
+    summary: Vault has gone sealed again (instance {{ $labels.instance }})
```

Comment:
I'm not sure about the phrasing of `has gone sealed again`. First, `has gone sealed` is strange, it's more that `Vault is sealed`. And the `again` can also not be true, on first startup, Vault will be sealed and it won't be "again".

Reply:
Improved